plugins/radare2/radare2_main.ml

```diff
@@ -31,8 +31,13 @@ let provide_radare2 file =
     let extract name json = Yojson.Basic.Util.member name json in
     try
      let symbol_list = Yojson.Basic.Util.to_list (R2.with_command_j "isj" file) in
+     let strip str = let open String in
+      match chop_prefix str ~prefix:"sym.imp." with
+      | Some str -> str
+      | None -> str 
+     in
      List.fold symbol_list ~init:() ~f:(fun () symbol -> 
-     accept (Yojson.Basic.Util.to_string (extract "name" symbol)) (Yojson.Basic.Util.to_int (extract "vaddr" symbol) |> Z.of_int)
+     accept (Yojson.Basic.Util.to_string (extract "name" symbol) |> strip) (Yojson.Basic.Util.to_int (extract "vaddr" symbol) |> Z.of_int)
```

Comment:
We're limiting our line length to 70 lines and using `ocp-indent` to indent the code. Could you please cleanup the code?